.circleci/config.yml

```diff
@@ -240,6 +240,63 @@ jobs:
               --key-file circle-sa-key.json \
               --promote
 
+  public-ui-build-test:
```

Comment:
This is unfortunately asymmetric to our API deploys (which pushes both api and public-api in sequence via one command). This is probably fine for now based on the requirements, but let me know if you tried the other way and if you hit issues with it.

Reply:
Yeah. I thought about that. I like the simple granularity of the ui stuff in circle .. For example , I don't want to run angular tests in public-ui now so I just removed that from the steps in public-ui-build-test... The benefits of combining the ui-build-test and public-ui-build-test doesn't seem worth the time to me right now.